load-tests/simple_producer_consumer/deployment/ingress.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
We need add the `trigger-load-test` and `process-load-test-results` scripts also